packaging/rpm/microshift.spec

```diff
@@ -244,8 +244,8 @@ install -m644 packaging/selinux/microshift.pp.bz2 %{buildroot}%{_datadir}/selinu
 install -d -m755 %{buildroot}%{_datadir}/microshift/functions
 install -p -m644 packaging/greenboot/functions.sh %{buildroot}%{_datadir}/microshift/functions/greenboot.sh
 
-install -d -m755 %{buildroot}%{_sysconfdir}/greenboot/check/required.d
-install -p -m755 packaging/greenboot/microshift-running-check.sh %{buildroot}%{_sysconfdir}/greenboot/check/required.d/40_microshift_running_check.sh
+install -d -m755 %{buildroot}%{_prefix}/lib/greenboot/check/required.d
```

Comment:
You should update the changelog section of this file, too.

Reply:
You're right. Updated